components/bl-multiple-bar-chart-component/src/validate.js

```diff
@@ -0,0 +1,21 @@
+export const validate = items => {
+  if (!items) {
+    return [];
+  }
+  
+  if (!items[0].hasOwnProperty('objectId')) {
+    return items.map(item => ({
+      ...item,
+      objectId: getId(),
+    }));
+  }
+    
+  return items;
+};
+
+const getId = () => {
```

Comment:
@Valodya I assume it's good candidate to share between all components.

Reply:
`BackendlessUI.UUID.short()`